src/widgets/modals/PatientHistory.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { getColumns, recordKeyExtractor, getItemLayout } from '../../pages/dataTableUtilities';
+import { DataTable, DataTableRow, DataTableHeaderRow } from '../DataTable';
+import { selectSortedPatientHistory } from '../../selectors/patient';
+import { PatientActions } from '../../actions/PatientActions';
+
+import { WHITE } from '../../globalStyles';
+
+const PatientHistory = ({ sortKey, onSortColumn, isAscending, data }) => {
+  const columns = React.useMemo(() => getColumns('patientHistory'), []);
+
+  const renderHeader = React.useCallback(
+    () => (
+      <DataTableHeaderRow
+        columns={getColumns('patientHistory')}
+        onPress={onSortColumn}
+        isAscending={isAscending}
+        sortKey={sortKey}
+      />
+    ),
+    [sortKey, isAscending]
+  );
+
+  const getCallback = React.useCallback(colKey => {
+    switch (colKey) {
```

Comment:
`case 'comingSoon'`?

Reply:
yeah - will be handling the `REPEAT` column